File: C/101.c
#include <stdlib.h>
#include <stdio.h>

int main(void)
{
    for (int multiplicand = 1; multiplicand <= 10; ++multiplicand)
    {
        for (int multiplicator = 1; multiplicator <= 10; ++multiplicator)
        {
            int result = multiplicator * multiplicand;
            fprintf(stdout, "%d x %d = %d\n", multiplicator, multiplicand, result);
        }
        fputc('\n', stdout);
    }

    return EXIT_SUCCESS;
}

File: Ruby/101.rb
#!/usr/bin/ruby

def main()
  multiplicand = 1
  while multiplicand <= 10
    multiplicator = 1
    while multiplicator <= 10
      result = multiplicator * multiplicand
      $stdout.print("%d x %d = %d\n" % [multiplicator, multiplicand, result])
      multiplicator += 1
    end
    $stdout.print("\n")
    multiplicand += 1
  end
end


main()

